src/main/java/org/folio/inventory/instanceingress/handler/CreateInstanceIngressEventHandler.java

```diff
@@ -0,0 +1,240 @@
+import static org.folio.inventory.dataimport.util.AdditionalFieldsUtil.reorderMarcRecordFields;
+import static org.folio.inventory.dataimport.util.MappingConstants.INSTANCE_REQUIRED_FIELDS;
+import static org.folio.inventory.dataimport.util.MappingConstants.MARC_BIB_RECORD_FORMAT;
+import static org.folio.inventory.dataimport.util.MappingConstants.MARC_BIB_RECORD_TYPE;
+import static org.folio.inventory.domain.instances.Instance.INSTANCE_TYPE_ID_KEY;
+import static org.folio.inventory.domain.instances.Instance.SOURCE_KEY;
+import static org.folio.rest.jaxrs.model.EntityType.MARC_BIBLIOGRAPHIC;
+import static org.folio.rest.jaxrs.model.InstanceIngressPayload.SourceType;
+import static org.folio.rest.jaxrs.model.Record.RecordType.MARC_BIB;
+import static org.folio.rest.jaxrs.model.Snapshot.Status.PROCESSING_IN_PROGRESS;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonObject;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.HttpStatus;
+import org.folio.MappingMetadataDto;
+import org.folio.inventory.common.Context;
+import org.folio.inventory.dataimport.cache.MappingMetadataCache;
+import org.folio.inventory.dataimport.handlers.actions.CreateInstanceEventHandler;
+import org.folio.inventory.dataimport.handlers.actions.PrecedingSucceedingTitlesHelper;
+import org.folio.inventory.dataimport.handlers.matching.util.EventHandlingUtil;
+import org.folio.inventory.dataimport.util.AdditionalFieldsUtil;
+import org.folio.inventory.dataimport.util.ValidationUtil;
+import org.folio.inventory.domain.instances.Instance;
+import org.folio.inventory.domain.instances.InstanceCollection;
+import org.folio.inventory.instanceingress.InstanceIngressEventConsumer;
+import org.folio.inventory.services.IdStorageService;
+import org.folio.inventory.storage.Storage;
+import org.folio.processing.exceptions.EventProcessingException;
+import org.folio.processing.mapping.defaultmapper.RecordMapper;
+import org.folio.processing.mapping.defaultmapper.RecordMapperBuilder;
+import org.folio.processing.mapping.defaultmapper.processor.parameters.MappingParameters;
+import org.folio.rest.jaxrs.model.InstanceIngressEvent;
+import org.folio.rest.jaxrs.model.ParsedRecord;
+import org.folio.rest.jaxrs.model.RawRecord;
+import org.folio.rest.jaxrs.model.Record;
+import org.folio.rest.jaxrs.model.Snapshot;
+
+public class CreateInstanceIngressEventHandler extends CreateInstanceEventHandler implements InstanceIngressEventHandler {
+
+  private static final Logger LOGGER = LogManager.getLogger(CreateInstanceIngressEventHandler.class);
+  private static final String BIBFRAME = " (bibframe)";
```

Comment:
No space before brackets
`private static final String BIBFRAME = "(bibframe)";`

Reply:
correct, fixed, thank you!